src/containers/UserMenu.jsx

```diff
@@ -121,6 +141,11 @@ export class UserMenu extends Component {
               {currentUser.email}
             </MenuItem>
             <Divider />
+            {window.MULTI_TENANT && isSuperAdmin ? (
```

Comment:
remove `window.MULTI_TENANT &&` -- anyone that's a superadmin can see this -- we don't need to turn it on.

Reply:
cool!